bindings/smtp/smtp.go

```diff
@@ -132,12 +141,13 @@ func (s *Mailer) parseMetadata(meta bindings.Metadata) (Metadata, error) {
 	smtpMeta.EmailBCC = meta.Properties["emailBCC"]
 	smtpMeta.EmailFrom = meta.Properties["emailFrom"]
 	smtpMeta.Subject = meta.Properties["subject"]
+	smtpMeta.Priority = defaultPriority
```

Comment:
Please, add a priority metadata support too, so it can be set at the configuration level as well. If not present, then default to this. Thanks for your contribution.